src/content/app/genome-browser/contexts/BrowserContext.tsx

```diff
@@ -0,0 +1,59 @@
+/**
```

Comment:
I was wondering which one to use i.e. Browser or GenomeBrowser. Are we going forward prefixing everything related to the genome browser as GenomeBrowser instead of Browser?

Reply:
✅

Reply:
> I was wondering which one to use i.e. Browser or GenomeBrowser. Are we going forward prefixing everything related to the genome browser as GenomeBrowser instead of Browser?

Preferably, yes.